climada/util/plot.py

```diff
@@ -472,3 +472,70 @@ def get_transformation(crs_in):
     except KeyError:
         units = '°'
     return crs_epsg, units
+
+
+def bar_plot(ax, data, colors=None, total_width=0.8, single_width=1, legend=True):
+    """Draws a bar plot with multiple bars per data point.
+    https://stackoverflow.com/questions/14270391/python-matplotlib-multiple-bars
```

Comment:
Just a detail, this works only for horizontal bar plots, right? (since below, x and y dimensions are explicitly treated.

Reply:
Do you think it would be useful to also have the option to make it y-bars instead of x-bars? I could add it.

Reply:
I think it would be a nice feature if it's not to much effort, yes

Reply:
Done :D.